src/pages/CalendarPage/CalendarPage.tsx

```diff
@@ -84,6 +94,8 @@ function UpdateAvailabilityModal({
                 />
             </Modal.Body>
             <Modal.Footer>
+                <Button onClick={() => onClickResetHandle()}>Reset</Button>
+                <Button onClick={() => onClickClearHandle()}>Clear</Button>
                 <Button
```

Comment:
In this case its sufficient to just pass a reference to the handlers rather than wrapping them in a function on lines 97, 98, and 101 since all we are looking to do is call the function, for example:

```
<Button onClick={onClickResetHandle}>Reset</Button>
```

If you just pass a reference to a handler function it will automatically call the function with whatever args are passed, in the case of the onClick it will call the onClickResetHandle with a mouse event which is why you can do the same for line 101.

Reply:
Do we need to define the handlers if they only call another function? Can we just keep it inline and call the function which the handler calls? Or is it cleaner to wrap it around defined handlers?

Reply:
I think its best practice to define the handlers because it makes it easier to refactor in the future. If we did something like - 

```
<Button onClick={() => setUserAvailabilityData(zeroState) }>Clear</Button>
```

This would work too but in the case where we have to do anything more than simply update the state with zeroState, say deal with the mouse event or update another state then it quickly gets out of hand.

Reply:
I see. Yeah, you're right it would be easier for a future refactor.